nautobot/extras/models/jobs.py

```diff
@@ -725,6 +725,8 @@ def linked_record(self):
     def get_absolute_url(self):
         return reverse("extras:jobresult", kwargs={"pk": self.pk})
 
+    # FIXME(jathan): This needs to go away. Need to think about that the impact
+    # will be in the JOB_RESULT_METRIC and how to compensate for it.
```

Comment:
@Kircheneer I would appreciate your thoughts on this comment and howe we might approach it.

Reply:
Where is the job status setting logic moving? Can we just move the metric creation there?